src/Components/Collections/CollectionsUtils/InputUtils/AddArray.js

```diff
@@ -71,7 +60,7 @@ const AddArray = ({childTypes, customTypes, dataTypes, isSet, identifier, parent
 
     const typeControls = (type, input) => {
         return type === 'nil' ? 'nil'
-            : type === 'str' ? (input[0]=='\''? `${input}`:`'${input}'`)
+            : type === 'str' ? (input[0]=='"'? `${input}`:`'${input}'`)
```

Comment:
I wonder if this is correct, if as a user you want to have the double quotes to be part of the string, is that still possible? And what if you have something like `"foo` ?

Reply:
Changed it back to single quotes.:)